dev/FileLogger.php

```diff
@@ -53,26 +55,26 @@ public function cleanLogs()
         return $logs;
     }
 
-    public function getLogs()
+    public function getLogs(): array
     {
         $filename = $this->getCSVFile();
         if (!is_file($filename)) {
             return [];
         }
         $csv = Reader::createFromStream(fopen($this->getCSVFile(), 'rb'));
         $csv->setDelimiter(';');
-        $csv = $csv->fetchAll();
-        return array_map(function ($line) {
-            $line[2] = json_decode($line[2], true);
+
+        return array_map(function (array $line): array {
```

Comment:
My suggestion here is place this in a private function with a descriptive name.

Reply:
Again, I'm not changing this as this is dev code. It functions well enough for now.